### DISCUSSION TITLE
Listing created via createDraftListing function is not editable

### DISCUSSION THREAD

**Discussion by TravnikovDev** [etsy/open-api#1296] [Q&A]
Hello,

I'm selling digital items on Etsy and trying to simplify the product creation process using the API. However, I keep encountering the same issue.

I am having problems with listings created via this particular endpoint: [https://openapi.etsy.com/v3/application/shops/{shop_id}/listings](https://openapi.etsy.com/v3/application/shops/%7Bshop_id%7D/listings)

I'm sending all the necessary data to the API, and it successfully creates a listing. I can edit a draft but am unable to publish it using the new Listing form. However, I am able to publish the listing using the dropdown menu.

As a result, I have a published listing, but every time I try to update it, I get a similar error:

```
There was a problem saving your changes. Contact Etsy Support and make sure to include this error code in your message:
Error code: EuXi_7433rfcELdvvZSQceyU2-cd
```

Interestingly, I can easily modify the listing via bulk menu options but not through the new listing form. This probably means that either the new form contains bugs or the API has issues with some parameters.

Another interesting fact: I can make a copy of the listing via the UI, and it works without issues.

Here's an example of the data I'm sending to the server:

```json
{
  "title": "Baby Fox SVG Bundle - 36 Designs - Nursery Decor - Cute Animal Clipart - Minimalist - Fairytale - Rustic Garden - Cricut and Laser Cutting",
  "description": "Add a touch of magic and charm to your ...",
  "tags": "Baby Fox Clipart,Nursery Decor SVG,Cute Animal Designs,Baby Fox Laser Files,Fox Cricut SVG,Whimsical Nursery,Fairytale Baby Fox,Rustic Clipart,SVG Bundle,Adorable Fox Designs,Baby Fox PNG,Creative Fox Clipart,Nursery Ar",
  "quantity": "999",
  "who_made": "i_did",
  "when_made": "2020_2024",
  "taxonomy_id": "6844",
  "price": "2.99",
  "is_supply": "false",
  "is_digital": "true",
  "type": "download",
  "shop_section_id": "40294382",
  "should_auto_renew": "true"
}
```

I tried to contact support but received only an automatic response.

Please help me figure out what's wrong.

**Comment by showtroylove** [comment]
I need more details to assist you properly. Have you created a listing directly using Etsy’s new listing form? If so, were you able to publish it successfully, as you mentioned?

**Reply by TravnikovDev** [reply]
I wrote all details into my question

> I am having problems with listings created via this particular endpoint: [openapi.etsy.com/v3/application/shops/{shop_id}/listings](https://openapi.etsy.com/v3/application/shops/%7Bshop_id%7D/listings)

**Comment by amitkumar-dev92** [comment]
Hello @TravnikovDev,
Did u provided `weight in lb[pound] (like weight=2lb)`
One of my client was also facing same issue, that time I tried to convert the weight in oz[ounce].
and the issue got resolved.
this is very strange, but it worked.

**Reply by TravnikovDev** [reply]
It's a digital item, so it doesn't have such fields as **weight**

**Comment by samborambo305** [comment]
@TravnikovDev Did you end up finding a solution to this? Our users are running into the same thing but we're unable to reliable reproduce it.

**Reply by TravnikovDev** [reply]
I just stopped using the Etsy API

**Comment by cyrilchandelier** [comment]
I'm also getting this error and have reported it to Etsy via their developer email support, I'm hoping they will answer eventually.

**Reply by lorenam83** [reply]
Did your description had emojis? 

I was recently having the same issues, i removed the emojis from the description sent to the API and published the listing, then tried editing it (to see if the error persisted) by adding back the emojis but now manually on the listing editor, and it worked snd saved the changes without issues.

hope this helps!
